nix/modules/holochain.nix

```diff
@@ -27,7 +27,7 @@
         OPENSSL_LIB_DIR = "${opensslStatic.out}/lib";
         OPENSSL_INCLUDE_DIR = "${opensslStatic.dev}/include";
 
-        buildInputs = (with pkgs; [ openssl opensslStatic sqlcipher ])
+        buildInputs = (with pkgs; [ openssl opensslStatic sqlcipher go ])
```

Comment:
if you add it here, you don't need to add it below in `release-automation`, because the below uses `holochain.nix` build inputs anyway.

Reply:
toolchains go only into the `nativeBuildInputs`. see https://nixos.org/manual/nixpkgs/stable/#var-stdenv-nativeBuildInputs